janusgraph-cql/src/main/java/org/janusgraph/diskstorage/cql/CQLStoreManager.java

```diff
@@ -185,6 +186,13 @@ public CQLStoreManager(final Configuration configuration) throws BackendExceptio
         fb.optimisticLocking(true);
         fb.multiQuery(false);
 
+        Boolean externalLocking = configuration.get(EXTERNAL_LOCKING);
+        LOGGER.info("External locking: {}", externalLocking);
+        if (externalLocking) {
+            // let JanusGraph core know that we'll handle lock on our own
+            fb.locking(true);
```

Comment:
please place with the other `fb.` setters and use `hasExternalLocking` instead of `true`

```
fb.locking(hasExternalLocking);
```

Reply:
Done